packages/openapi-request-validator/index.ts

```diff
@@ -733,3 +736,14 @@ function transformOpenAPIV3Definitions(schema) {
   recursiveTransformOpenAPIV3Definitions(res);
   return res;
 }
+
+function getCaseInsensitiveHeaders(
+  requestHeaders: { [key: string]: any },
+  header: string
+) {
+  const pattern = new RegExp('(' + header + ')', 'i');
```

Comment:
can you use `.toLowerCase` instead of `RegExp`?

Reply:
That would restrict the code to only search for lowercase headers. But since used in that manner that seems better. Will make the change